rules/sonic-fips.mk

```diff
@@ -0,0 +1,41 @@
+# fips packages
+
+FIPS_VERSION = 0.1
+FIPS_OPENSSH_VERSION = 8.4p1-5+fips
+FIPS_PYTHON_MAIN_VERSION = 3.9
+FIPS_PYTHON_VERSION = 3.9.2-1+fips
+FIPS_GOLANG_MAIN_VERSION = 1.15
+FIPS_GOLANG_VERSION = 1.15.15-1~deb11u2+fips
+BLDENV ?= $(shell lsb_release -cs)
```

Comment:
>BLDENV [](http://example.com/codeflow?start=0&length=6)

This is already used by slave.mk. Do you reassign?

Reply:
The variable is not exported in slave.mk, we use the same way with the slave.mk. May be better to export it in slave.mk.

Reply:
If they are the same thing, better to export in slave.mk

Reply:
The `BLDENV` variable should be available in this context. It's being used in `rules/snmpd.mk` for example, so the export shouldn't be needed.

Reply:
Fixed, thank.